integration-tests/tests/example-page.cy.ts

```diff
@@ -70,11 +71,11 @@ if (!Cypress.env('OPENSHIFT_CI') || Cypress.env('PLUGIN_TEMPLATE_PULL_SPEC')) {
     });
 
     it('Verify the url', () => {
+      cy.get('[data-quickstart-id="qs-nav-home"]').click();
```

Comment:
> It's unclear to me what this is doing... Is it clicking on a quick start?

No, clicking on the Home side nav. Used the `data-quickstart-id` instead since there is no `data-test` attribute  in the Home button. Can add  the `data-test` to console or use css selector if it doesn't look good to you.